test/integration/ruby_uninstall/inspec.yml

```diff
@@ -0,0 +1,9 @@
+name: rbenv
+title: rbenv profile
+maintainer: Sous Chefs
+copyright: Webb Agile Solutions Ltd.
+license: Apache-2.0
+summary: Verifies rbenv is installed correctly
```

Comment:
Also the summary is the same everywhere, is that intended?

Reply:
I was mostly being lazy with the description, if you want to expand upon it. Feel free to do it in another PR 👍